SecretManagement.KeePass.Extension/Tests/Test-SecretVault.Tests.ps1

```diff
@@ -102,29 +142,509 @@ InModuleScope -ModuleName 'SecretManagement.KeePass.Extension' {
                     ModuleName      = $ModulePath
                     PassThru        = $true
                     VaultParameters = @{
-                        Path = $VaultPath
+                        Path              = $VaultPath
                         UseMasterPassword = $true
                     }
                 }
                 Microsoft.PowerShell.SecretManagement\Register-SecretVault @RegisterSecretVaultPathOnlyParams | Out-Null
 
-                Mock -Verifiable -CommandName 'Get-Credential' -MockWith {$VaultMasterKey}
+                Mock -Verifiable -CommandName 'Get-Credential' -MockWith { $VaultMasterKey }
             }
             AfterAll {
                 try {
                     Microsoft.PowerShell.SecretManagement\Get-SecretVault -Name $VaultName -ErrorAction SilentlyContinue | Microsoft.PowerShell.SecretManagement\Unregister-SecretVault -ErrorAction SilentlyContinue
                 } catch [system.Exception] { }
             }
-                It "should not have a variable 'Vault_$($VaultName)'" {
+            It "should not have a variable 'Vault_$($VaultName)'" {
                 { (Get-Variable -Name "Vault_$VaultName" -Scope Script).Value } | Should -Throw
```

Comment:
-Throw should be relatively explicit to the expected exception, in this case: `| Should -Throw 'Cannot find a variable with the name*'`